packages/editor/src/editor-styles/ast/parse.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @wordpress/no-unused-vars-before-return */
```

Comment:
I see you gave up here :)

Reply:
Maybe we should update it. I just wasn't sure if we wanted to commit to a fork.

Reply:
If this is a fork, then I think it's fine to leave it as is.